Module not found: Error: Can't resolve 'camunda-dmn-moddle/resources/camunda' Angular 9

__Describe the Bug__

Trying to serve an angular 9 application with Dmn-js but getting error:
```
ERROR in ./node_modules/dmn-js-shared/lib/base/Manager.js
Module not found: Error: Can't resolve 'camunda-dmn-moddle/resources/camunda' in 'my-project-directory\node_modules\dmn-js-shared\lib\base'
```

I was able to do a temporary fix in `Manager.js` by changing line 25 from:
```
import CamundaModdle from 'camunda-dmn-moddle/resources/camunda';
```
to: 
```
import * as CamundaModdle from 'camunda-dmn-moddle/resources/camunda.json';
```

I don't believe this is the correct fix but it works for me for now. Please note I am a java developer and I am still learning typescript/angular 9. 

__Steps to Reproduce__

Steps to reproduce the behavior:

clone repo: https://github.com/alevyKorio/Angular9Dmnjs.git
attempt command: `npm run start`

__Expected Behavior__

To be able to serve.

__Environment__

dmn-js 8.3.0

Angular CLI: 9.1.7
Node: 10.15.3
OS: win32 x64

Angular: 9.1.9
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.901.7
@angular-devkit/build-angular     0.901.7
@angular-devkit/build-optimizer   0.901.7
@angular-devkit/build-webpack     0.901.7
@angular-devkit/core              9.1.7
@angular-devkit/schematics        9.1.7
@angular/cli                      9.1.7
@ngtools/webpack                  9.1.7
@schematics/angular               9.1.7
@schematics/update                0.901.7
rxjs                              6.5.5
typescript                        3.8.3
webpack                           4.42.0

--- a/packages/dmn-js-shared/src/base/Manager.js
+++ b/packages/dmn-js-shared/src/base/Manager.js
@@ -2,7 +2,7 @@ import EventBus from 'diagram-js/lib/core/EventBus';
 
 import DmnModdle from 'dmn-moddle';
 
-import CamundaModdle from 'camunda-dmn-moddle/resources/camunda';
+import CamundaModdle from 'camunda-dmn-moddle/resources/camunda.json';
 
 import {
   domify,

